src/vertex_array_object.rs

```diff
@@ -376,6 +376,14 @@ fn vertex_binding_type_to_gl(ty: AttributeType) -> (gl::types::GLenum, gl::types
         AttributeType::U32U32 => (gl::UNSIGNED_INT, 2, 1),
         AttributeType::U32U32U32 => (gl::UNSIGNED_INT, 3, 1),
         AttributeType::U32U32U32U32 => (gl::UNSIGNED_INT, 4, 1),
+        AttributeType::I64 => (gl::INT, 1, 1),
+        AttributeType::I64I64 => (gl::INT, 2, 1),
+        AttributeType::I64I64I64 => (gl::INT, 3, 1),
+        AttributeType::I64I64I64I64 => (gl::INT, 4, 1),
+        AttributeType::U64 => (gl::UNSIGNED_INT, 1, 1),
+        AttributeType::U64U64 => (gl::UNSIGNED_INT, 2, 1),
+        AttributeType::U64U64U64 => (gl::UNSIGNED_INT, 3, 1),
+        AttributeType::U64U64U64U64 => (gl::UNSIGNED_INT, 4, 1),
```

Comment:
I'm not sure what `gl::` type should be used here, but I'm pretty sure it's not these ones. I tried `gl::LONG` and `gl::UNSIGNED_LONG` but they weren't right. I search through the gl_bindings.rs file in the target directory, and couldn't figure it out.